app/assets/javascripts/leaflet.map.js

```diff
@@ -55,6 +55,13 @@ L.OSM.Map = L.Map.extend({
 
     this.dataLayer = new L.OSM.DataLayer(null);
     this.dataLayer.options.code = 'D';
+    
+    this.gpsLayer = new L.OSM.GPS({
+      attribution: copyright,
+      code: "G",
+      keyid: "gps",
```

Comment:
Only base layers need `keyid` as there is no support for showing keys for overlays.

The `code` member is fine, but it needs code adding at https://github.com/openstreetmap/openstreetmap-website/blob/master/app/assets/javascripts/index.js#L160 to actually add the layer if that code is present when the page is loaded.